tests/components/nest/test_config_flow_sdm.py

```diff
@@ -81,27 +122,32 @@ async def async_oauth_flow(self, result):
         with patch(
             "homeassistant.components.nest.async_setup_entry", return_value=True
         ) as mock_setup:
-            await self.hass.config_entries.flow.async_configure(result["flow_id"])
+            await self.hass.config_entries.flow.async_configure(
+                result["flow_id"], user_input
+            )
             assert len(mock_setup.mock_calls) == 1
```

Comment:
We should block till done here after finishing the flow so we make sure the patch is in place until all tasks are done.